sources/packages/web/src/components/aest/students/bypassedRestrictions/HistoryBypassedRestrictions.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+  <v-card>
+    <v-container>
+      <body-header title="History of Bypassed Restrictions" class="m-1">
+        <template #actions>
+          <check-permission-role :role="Role.StudentAddRestriction">
+            <template #="{ notAllowed }">
+              <v-btn
+                @click="addBypassedRestriction"
```

Comment:
Please remove the click attribute and the method `addBypassedRestriction` here. The upcoming ticket will handle it.

Reply:
Removed.